### ISSUE TITLE
train the model, but I get an empty all_models folder

### ISSUE THREAD

**Issue by YuxiuLiu** [jamesgleave/DD_protocol#1]
Hi, I run the script simple_job_models_manual.py, and set the arguments as follows:
--iteration_no 1 --morgan_directory /data3/liuyx/DD_test/morgan_lib --file_path /data3/liuyx/DD_test/project --number_of_hyp 16 --total_iterations 4 --is_last False --number_mol 8000 --percent_first_mols 20 --percent_last_mols 0.1 --recall 0.90 
I want to test the model, so I chose smaller scale of library which contain the training set of 8734 molecules (score number in training_labels.txt), test set of 8674 molecules (score number in testing_labels.txt) and valid set of 8760 molecules(score number in validation_labels.txt) . However, after I ran the script, the created folder all_models is empty and no hyperparameter_morgan_with_freq_v3.txt file is obtained. Can you help me about this problem? Thanks a lot.

**Comment by jamesgleave** [comment]
Hi, thanks for reaching out. It seems like `progressive_docking.py`, which is run by `simple_job_models_manual.py`, did not execute properly. `progressive_docking.py` creates and populates the `all_models` directory and the `hyperparameter_morgan_with_freq_v3.txt` file near the beginning and at the end of execution, respectfully. Would you be able to provide us with one output file for each of the `simple_job_models_manual.py` and `progressive_docking.py` scripts?

**Comment by YuxiuLiu** [comment]
Hi professor, thanks a lot for your answer. And now we solve the previous
problem, but we met a problem again when we trained the model using the
script progressive_docking.py script. We use the arguments as
follows: python -u progressive_docking.py -os 10 -bs 256 -num_units 1500
-dropout 0.2 -learn_rate 0.0001 -bin_array 2 -wt 2 -cf -0.857812344519887
-rec 0.9 -n_it 1 -t_mol 0.01999 --data_path
/data3/liuyx/DD_test_2/protein_test --save_path
/data3/liuyx/DD_test_2/protein_test -n_mol 1928. And the problem we met was
summarized in enclosed document. Can you help me about this? Thanks a lot.
we apply glide sp to dock the sampled molecules. 2000 molecules for
testing, training and validation respectively. And 20000 molecules for
total database.

James Gleave ***@***.***> 于2022年4月20日周三 00:33写道：

**Comment by YuxiuLiu** [comment]
Hi professor, I'm sorry to bother you again, we have solved the problem
mentioned yesterday. And we trained the model successfully, then we used
the hyperparameter_result_evaluation.py script to perform the grid search,
however we met the problem summarized in enclosed document.The arguments we
used as follows:
python hyperparameter_result_evaluation.py --n_iteration 1 --data_path
/data3/liuyx/DD_test_2/protein_test --morgan_directory
/data3/liuyx/DD_test_2/library_prepared_fp --number_mol 1900 --recall 0.90
Thanks a lot for your sincere help.

yuxiu liu ***@***.***> 于2022年4月21日周四 18:18写道：

**Comment by jamesgleave** [comment]
Hi, no problem; I am glad the prior issues got resolved. Unfortunately, I do not see an attached document. Can you send us a stack trace or some more details regarding the problem? Thanks!

**Comment by YuxiuLiu** [comment]
Hi professor, thanks a lot for your reply, and we've solved all
the problems and run the Deep Docking successfully.
With best regards!
Yuxiu Liu

James Gleave ***@***.***> 于2022年4月23日周六 05:35写道：
